pyopencl/tools.py

```diff
@@ -286,8 +286,15 @@ def __str__(self):
 
             arg_values.append(tuple(arg_dict[name] for name in arg_names))
 
+    def idfn(val):
+        if isinstance(val, cl.Platform):
+            # Don't show address, so that parallel test collection works
+            return f"<pyopencl.Platform '{val.name}'>"
```

Comment:
This seems important, but I don't fully understand it. What's the connection to parallel test collection?

Reply:
OIC: https://github.com/inducer/pyopencl/runs/871508867

Reply:
Thanks!